### ISSUE TITLE
ALERT !!

### ISSUE THREAD

**Issue by And42** [Nikitinho/Risk-Manager-iOS#3]
Can't  see the point of the app...

**Comment by Nikitinho** [comment]
Hello, And42. Why are you so categorical? Nowdays even useless apps for terraria become very successful projects.

**Comment by And42** [comment]
Yeah, kind of... but really, just a notes app, what is so great about it? What are the unique features?

**Comment by Nikitinho** [comment]
The app is under development and a lot of features will be introduced soon. Just wait a bit :wink:

**Comment by And42** [comment]
Thx for the answer but can you please say what features will it have? Just thinking about telling my manager about it and want to know as much as possible

**Comment by Nikitinho** [comment]
Unfortunately, I can't put my cards on the table. I'm the only one developer of this app and if everybody knows what unique features I'm going to provide, I won't have enough time for implementation. Main competitors will not wait...

**Comment by And42** [comment]
Oh, understand. So then good luck, hope everything will be done in time ;)
